typescript-modal/main.ts

```diff
@@ -0,0 +1,17 @@
+const $openModal = document.querySelector('.open-modal');
+const $dismissModal = document.querySelector('.dismiss-modal');
+const $dialog = document.querySelector('dialog');
+
+if (!$openModal) throw new Error('Error1');
+if (!$dismissModal) throw new Error('Error2');
+if (!$dialog) throw new Error('Error3');
```

Comment:
These aren't really good error messages. If you got an error that says "Error1" would you have any idea what that means? Your error messages should be more descriptive of what actually went wrong.

Look at the snippet below, this would be a better error message, because it actually tells me what went wrong.
```
if (!$openModal) throw new Error('$openModal query failed');
```